Source/control.cpp

```diff
@@ -368,7 +368,8 @@ void DrawSpellList(CelOutputBuffer out)
 			c = SPLICONLAST + 2;
 			break;
 		}
-		for (spl = 1, j = 1; j < MAX_SPELLS; spl <<= 1, j++) {
+		Sint32 j = SPL_FIREBOLT;
+		for (spl = 1; j < MAX_SPELLS; spl <<= 1, j++) {

```

Comment:
Where's `j` coming from now? Can we really remove its initialization here?

Reply:
Look at the line above...

Reply:
Missed it for some reason. Thanks.